Looks like it missing a Makefile:

```
building path(s) ‘/nix/store/zxmnqc34yavbx9fg0p9dwyfcvlmcclvi-xpdf-4.00’
unpacking sources
unpacking source archive /nix/store/hpkwr44b5rhlcnhikhw9y0a7ll0cccxk-xpdf-4.00.tar.gz
source root is xpdf-4.00
setting SOURCE_DATE_EPOCH to timestamp 1502234570 of file xpdf-4.00/xpdf/pdftotext.cc
patching sources
configuring
no configure script, doing nothing
building
no Makefile, doing nothing
installing
install flags:
 install
make: *** No rule to make target 'install'.  Stop.
```

--- a/pkgs/applications/misc/xpdf/default.nix
+++ b/pkgs/applications/misc/xpdf/default.nix
@@ -1,9 +1,9 @@
 { enableGUI ? true, enablePDFtoPPM ? true, useT1Lib ? false
-, stdenv, fetchurl, zlib, libpng, xlibsWrapper ? null, motif ? null, freetype ? null, t1lib ? null
-, base14Fonts ? null
+, stdenv, fetchurl, zlib, libpng, freetype ? null, t1lib ? null
+, cmake, qtbase ? null
 }:
 
-assert enableGUI -> xlibsWrapper != null && motif != null && freetype != null;
+assert enableGUI -> qtbase != null && freetype != null;
 assert enablePDFtoPPM -> freetype != null;
 assert useT1Lib -> t1lib != null;
 
@@ -17,8 +17,12 @@ stdenv.mkDerivation {
     sha256 = "1mhn89738vjva14xr5gblc2zrdgzmpqbbjdflqdmpqv647294ggz";
   };
 
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = ["-DSYSTEM_XPDFRC=/etc/xpdfrc" "-DA4_PAPER=ON"];
+
   buildInputs = [ zlib libpng ] ++
-    stdenv.lib.optionals enableGUI [xlibsWrapper motif] ++
+    stdenv.lib.optional enableGUI qtbase ++
     stdenv.lib.optional useT1Lib t1lib ++
     stdenv.lib.optional enablePDFtoPPM freetype;
 
@@ -26,14 +30,6 @@ stdenv.mkDerivation {
   CXXFLAGS = "-O2 -fpermissive";
 
   hardeningDisable = [ "format" ];
-
-  configureFlags = "--enable-a4-paper";
-
-  postInstall = stdenv.lib.optionalString (base14Fonts != null) ''
-    substituteInPlace $out/etc/xpdfrc \
-      --replace /usr/local/share/ghostscript/fonts ${base14Fonts} \
-      --replace '#fontFile' fontFile
-  '';
 
   meta = {
     homepage = http://www.foolabs.com/xpdf/;

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -11098,7 +11098,7 @@ with pkgs;
   dex-oidc = callPackage ../servers/dex { };
 
   dgraph = callPackage ../servers/dgraph { };
-    
+
   dico = callPackage ../servers/dico { };
 
   dict = callPackage ../servers/dict {
@@ -16924,9 +16924,7 @@ with pkgs;
 
   apvlv = callPackage ../applications/misc/apvlv { };
 
-  xpdf = callPackage ../applications/misc/xpdf {
-    base14Fonts = "${ghostscript}/share/ghostscript/fonts";
-  };
+  xpdf = libsForQt5.callPackage ../applications/misc/xpdf { };
 
   xkb_switch = callPackage ../tools/X11/xkb-switch { };
 
@@ -18684,7 +18682,7 @@ with pkgs;
   hplip_3_15_9 = callPackage ../misc/drivers/hplip/3.15.9.nix { };
 
   hplipWithPlugin_3_15_9 = hplip_3_15_9.override { withPlugin = true; };
-  
+
   epkowa = callPackage ../misc/drivers/epkowa { };
 
   illum = callPackage ../tools/system/illum { };

